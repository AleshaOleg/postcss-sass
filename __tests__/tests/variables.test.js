const getPostCssTreeFromSass = require('../../helpers/getPostCssTreeFromSassTree');

it('variables.sass', function () {
    expect(getPostCssTreeFromSass('variables')).toMatchSnapshot();
});
