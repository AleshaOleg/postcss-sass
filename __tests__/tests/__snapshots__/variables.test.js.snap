// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`variables.sass 1`] = `
Object {
  "nodes": Array [
    Object {
      "prop": "$red",
      "raws": Object {
        "before": "",
        "between": ": ",
        "semicolon": false,
      },
      "source": Object {
        "end": Object {
          "column": 9,
          "line": 1,
        },
        "input": Input {
          "css": "$red: red

a
    color: $red
",
          "id": "<input css 1>",
        },
        "start": Object {
          "column": 1,
          "line": 1,
        },
      },
      "type": "decl",
      "value": "red",
    },
    Object {
      "nodes": Array [
        Object {
          "prop": "color",
          "raws": Object {
            "before": "
    ",
            "between": ": ",
            "semicolon": false,
          },
          "source": Object {
            "end": Object {
              "column": 15,
              "line": 4,
            },
            "input": Input {
              "css": "$red: red

a
    color: $red
",
              "id": "<input css 1>",
            },
            "start": Object {
              "column": 5,
              "line": 4,
            },
          },
          "type": "decl",
          "value": "$red",
        },
      ],
      "raws": Object {
        "before": "

",
        "between": "",
      },
      "selector": "a",
      "source": Object {
        "end": Object {
          "column": 16,
          "line": 4,
        },
        "input": Input {
          "css": "$red: red

a
    color: $red
",
          "id": "<input css 1>",
        },
        "start": Object {
          "column": 1,
          "line": 3,
        },
      },
      "type": "rule",
    },
  ],
  "raws": Object {
    "before": "",
    "semicolon": undefined,
  },
  "source": Object {
    "end": Object {
      "column": 16,
      "line": 4,
    },
    "input": Input {
      "css": "$red: red

a
    color: $red
",
      "id": "<input css 1>",
    },
    "start": Object {
      "column": 1,
      "line": 1,
    },
  },
  "type": "root",
}
`;
